java-frontend/src/test/java/org/sonar/java/JavaFrontendTest.java

```diff
@@ -144,7 +152,7 @@ void scanning_empty_project_should_be_logged_in_file_by_file() throws Exception
     settings.setProperty("sonar.java.fileByFile", "true");
     scan(settings, SONARQUBE_RUNTIME, Collections.emptyList());
 
-    assertThat(logTester.logs(LoggerLevel.INFO)).containsExactly(
```

Comment:
Can we not keep the `containsExactly` assertion, given that it is stronger than `contains`?